README.md

```diff
@@ -1,33 +1,18 @@
 ## Codecademy React Challenge Project
 
-This project will ask the learner to design an appointment app that resembles those of medical offices like LabCorp and Quest Diagnostics. The app will allow users logged in as Providers to add appointment availability and users logged in as Patients" to reserve an available appointment. 
+#### Notes 2/16/21
+- react-datepicker: I was able to grab the css from *node_modules* for this component and put it in **index.html**. Will the LE have access to it the react-datepicker component though? I will create a date validation field if necessary.
+- I am curious how css is going to work. I feel I will need to describe how to include CSS in the project description. The design with class names will be needed before finalizing the instructions, correct?
```

Comment:
See this thread https://codecademy.slack.com/archives/C01KHH5NW05/p1613586814003100